package.json

```diff
@@ -28,7 +28,7 @@
     "start": "nest start",
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
-    "start:prod": "node --max-old-space-size=4096 dist/main",
+    "start:prod": "test -f dist/main.js && node --max-old-space-size=4096 dist/main || node --max-old-space-size=4096 dist/src/main",
```

Comment:
`node --max-old-space-size=4096 dist/src/main` for local start?

Reply:
Yes